src/model_repository_manager.cc

```diff
@@ -359,7 +360,7 @@ class ModelRepositoryManager::ModelLifeCycle {
   // not specified in the load will be unloaded after the load is finished.
   Status AsyncLoad(
       const std::string& repository_path, const std::string& model_name,
```

Comment:
Is `repository_path` still being used? I don't think we need to keep it given `model_path`

Reply:
Thanks, removed repository_path throughout for loading (and ModelInfo). Looks like there was a FIXME in there to get rid of it.